api/server/server.go

```diff
@@ -324,7 +326,7 @@ func getImagesJSON(eng *engine.Engine, version version.Version, w http.ResponseW
 		Filters: r.Form.Get("filters"),
 		// FIXME this parameter could just be a match filter
 		Filter: r.Form.Get("filter"),
-		All:    toBool(r.Form.Get("all")),
+		All:    boolValue(r, "all"),
```

Comment:
parsing integer hadn't been needed until now in the api and I wanted to be consistent. I don't mind to open a separated PR, but I don't think it's too much of a risk.